src/boot/main.c

```diff
@@ -128,6 +128,24 @@ void create_thread(OSThread *thread, OSId id, void (*entry)(void *), void *arg,
 extern void func_sh_802f69cc(void);
 #endif
 
+void change_vi(OSViMode *mode, int width, int height){
```

Comment:
uh any reason to move this function?

Reply:
iirc this was to avoid redefining it at the top of the function since it's not in a header I don't think

Reply:
wouldnt it be easier and less conflicting to just add it in the header?

Reply:
I generally don't like externing functions whenever it's not necessary but I suppose there isn't too much harm done here if we do.